concent_api/verifier/utils.py

```diff
@@ -87,7 +98,7 @@ def unpack_archive(file_path):
 
 def get_files_list_from_archive(file_path):
     """ Returns list of files from given zip archive. """
-    return zipfile.ZipFile.namelist(file_path)
+    return zipfile.ZipFile(file_path).namelist()
```

Comment:
What is the difference between those two ?

Reply:
the first one doesn't work ;) - raises some Exception

Reply:
Really ? I remember testing this part in context of memory used and it worked fine then. But if you say so and it works after change then it is ok.